exporter/awss3exporter/README.md

```diff
@@ -38,7 +38,9 @@ The following exporter configuration parameters are supported.
 | `compression`             | should the file be compressed                                                                                                              | none                                        |
 | `sending_queue`           | [exporters common queuing](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)          | disabled                                    |
 | `timeout`                 | [exporters common timeout](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)          | 5s                                          |
-
+| `retry_mode`              | The retryer implementation, the supported values are "standard"(default) and "adaptive"(experimental)                                      | none                                        |
+| `retry_max_attempts`      | The max number of attempts for retrying a request if the `retry_mode` is set                                                               | 3                                           |
```

Comment:
I thought it was standard if not set?

Reply:
Yup it was a mistake, I will address this together with other changes later on.